### ISSUE TITLE
Partioned CBC output cannot be merged when boundary condition not present in every partition

### ISSUE THREAD

**Issue by Manangka** [Deltares/imod-python#683]
In GitLab by @JoerivanEngelen on Dec 7, 2023, 15:09

When one boundary condition is missing in one partition, it is dropped from the model (logically). This causes certain variables (e.g. ```cbc["wel"]```) to be missing in a certain partition. When merging, this causes xugrid to throw a clear error: 

```
ValueError: These variables are present in some partitions, but not in others: {'well'}
```

**Comment by Manangka** [comment]
In GitLab by @JoerivanEngelen on Dec 7, 2023, 15:10

marked this issue as related to #658

**Comment by luitjansl** [comment]
this issue was solved along with #846  in https://github.com/Deltares/imod-python/pull/881
